implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -236,6 +247,22 @@ async fn run_foreground_node(
         .await?;
     }
 
+    if let Some(authority_identities) = &cmd.authority_identities {
+        for identity in authority_identities.iter() {
+            let identity_as_bytes = match hex::decode(identity) {
```

Comment:
I think it would be a good to use a [value parser](https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#validated-values), and have `pub authority_identities: Option<Vec<Authorities>>` directly on the `CreateCommand. Then this parser can be reused for other commands.